text/0003-content_addressed_file_system.md

```diff
@@ -0,0 +1,143 @@
+- Feature Name: content_addressed_file_system
+- Start Date: 2017-08-03
+- RFC PR: [#3](https://github.com/qri-io/rfcs/pull/3)
+- Issue: NA
+
+# Summary
+[summary]: #summary
+
+Content-Addressed File System (cafs) is a generalized interface for working with 
+filestores that determine names content based on the content itself, usually
+through some sort of hashing function.
+Examples of a content-addressed file systems include git, bittorrent, IPFS, 
```

Comment:
should read: "Examples of content-addressed file systems include" (take out the 'a')